tests/unit/serve_test.ts

```diff
@@ -741,7 +741,7 @@ Deno.test({ permissions: { net: true } }, async function httpServerPort0() {
 });
 
 Deno.test(
-  { permissions: { net: true } },
+  { permissions: { net: true }, ignore: Deno.build.os !== "windows" },
```

Comment:
I think we should change the regexp below to match both patterns instead of ignoring the test case in non-windows.